integration-test/src/main/java/com/sequenceiq/it/cloudbreak/dto/database/RedbeamsDatabaseTestDto.java

```diff
@@ -0,0 +1,153 @@
+package com.sequenceiq.it.cloudbreak.dto.database;
+
+import java.util.List;
+// import java.util.function.BiConsumer;
+// import java.util.function.Function;
+import java.util.stream.Collectors;
+
+// import javax.ws.rs.WebApplicationException;
+
+// import com.sequenceiq.it.IntegrationTestContext;
+import com.sequenceiq.it.cloudbreak.dto.DeletableRedbeamsTestDto;
+// import com.sequenceiq.it.cloudbreak.Assertion;
+// import com.sequenceiq.it.cloudbreak.GherkinTest;
+import com.sequenceiq.it.cloudbreak.Prototype;
+import com.sequenceiq.it.cloudbreak.RedbeamsClient;
+import com.sequenceiq.it.cloudbreak.context.TestContext;
+import com.sequenceiq.it.cloudbreak.util.ResponseUtil;
+import com.sequenceiq.redbeams.api.endpoint.v4.database.DatabaseV4Endpoint;
+import com.sequenceiq.redbeams.api.endpoint.v4.database.request.DatabaseV4Request;
+import com.sequenceiq.redbeams.api.endpoint.v4.database.responses.DatabaseV4Response;
+
+@Prototype
+public class RedbeamsDatabaseTestDto extends DeletableRedbeamsTestDto<DatabaseV4Request, DatabaseV4Response, RedbeamsDatabaseTestDto, DatabaseV4Response> {
+
+    public static final String DATABASE = "DATABASE";
+
+    RedbeamsDatabaseTestDto(String newId) {
+        super(newId);
+        setRequest(new DatabaseV4Request());
+    }
+
+    RedbeamsDatabaseTestDto() {
+        this(DATABASE);
+    }
+
+    public RedbeamsDatabaseTestDto(TestContext testContext) {
+        super(new DatabaseV4Request(), testContext);
+    }
+
+    public RedbeamsDatabaseTestDto(DatabaseV4Request request, TestContext testContext) {
+        super(request, testContext);
+    }
+
+    // @Override
+    // public void cleanUp(TestContext context, RedbeamsClient redbeamsClient) {
+    //     LOGGER.info("Cleaning up resource with name: {}", getName());
+    //     try {
+    //         redbeamsClient.getEndpoints().databaseV4Endpoint().deleteByName(cloudbreakClient.getWorkspaceId(), getName());
+    //     } catch (WebApplicationException ignore) {
+    //         LOGGER.warn("Cleaning up resource failed", ignore);
+    //     }
+    // }
```

Comment:
Is this accidentally has been left behind or really necessary be here as commented out part?

Reply:
I can remove it, but maybe it will be needed soon. A problem with these tests is that they leave resources behind, but I haven't figured out how to clean them up yet. This looks like the code to do it.